README.md

```diff
@@ -90,7 +90,7 @@ circ.measure(0, 0)
 circ.measure(1, 1)
 
 # Default 6-qubit QPU with the ratio of memory dissipation rates set to
-# k1/k2=1e-5 and cat amplitude alpha set to 4.
+# k1/k2=1e-5 and cat size average_nb_photons set to 16.
```

Comment:
```suggestion
# k1/k2=1e-5 and cat size, average_nb_photons, set to 16.
```

Feels weird to read with no interruption. Wdyt?

Reply:
Great, thanks!